sample-new-architecture/src/Screens/GesturesTracingScreen.tsx

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { StyleSheet, Text, View } from 'react-native';
+import { Gesture, GestureDetector } from 'react-native-gesture-handler';
+import { getCurrentHub, Scope, traceGesture } from '@sentry/react-native';
+
+const GesturesTracingScreen = () => {
+  const gesture = Gesture.Pinch()
+    .onBegin(() => {
+      console.log('Pinch gesture begin');
+    })
+    .onEnd(() => {
+      console.log('Pinch gesture end');
+      startExampleSpan();
```

Comment:
Should the example start during `onBegin` and end during `onEnd`? its confusing right now.

Reply:
I wanted to showcase that you can start a fetch or something and the Sentry will trace it.

So the end of the gesture triggers it and then you can see in Sentry what happened after that.

Maybe I should rename the `startExampleSpan` function to be more clear to something like `fetchExampleData`.